### ISSUE TITLE
Slide button component - create quo2 component

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#15870]
Create the quo2 component Slide button as per designs:

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=19085-205591&t=bJcZWXGaearKHzum-0

Should be implemented according to best practices
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Include component spec tests
https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md

Communicate with design team before starting to make sure there is no unknown requirements

Please liase with @J-Son89 with regards of the API.

Blur variant is out of scope.

**Comment by souravbaranwal** [comment]
Hi @J-Son89 ,
I will be working on this issue.

**Comment by flexsurfer** [comment]
implemented here https://github.com/status-im/status-mobile/pull/16259
